src/community/geopkg/src/main/resources/org/geoserver/geopkg/wps/owsContextRequest.ftl

```diff
@@ -0,0 +1,65 @@
+{
+    "type":"FeatureCollection",
```

Comment:
This is a very interesting pull requests, is this documenting the source of the geopackage?

Reply:
It's a OWS context, in JSON format. IN particular, it's provenance, provides the original request that generated the GeoPackage.